05_node_operations.asciidoc

```diff
@@ -72,134 +72,134 @@ in the cloud. Nonetheless, if running your own server is not an option, by all m
 
-The fastest SSDs available today use the NVMe interface. The NVMe SSDs are faster in high end machines, but also more costly.
+The fastest SSDs available today use the NVMe interface. The NVMe SSDs are faster in high-end machines, but also more costly.
 Traditional SATA-based SSDs are cheaper, but not as fast. SATA SSDs perform sufficiently well for your node setup.
 Smaller computers might not be able to take advantage of NVMe SSDs.
 For example, the Raspberry Pi 4 cannot benefit from them because of the limited bandwidth of its USB port.
 
-To choose the size, let's look at the Bitcoin blockchain. As of January 2021 its size is 324GB including the transaction index and grows by roughly 60GB/year. If you want to have some margin available for future growth or to install other data on your node, purchase at least a 512GB drive or better yet a 1TB drive.
+To choose the size, let's look at the Bitcoin blockchain. As of April 2021, its size is 338GB including the transaction index and grows by roughly 60GB/year. If you want to have some margin available for future growth or to install other data on your node, purchase at least a 512GB drive or better yet a 1TB drive.
 
 === Using an installer or helper
 
 Installing a Lightning node or a Bitcoin node may be daunting if you are not familiar with a command-line environment. Luckily, there are a number of projects that make "helpers", i.e. software that installs and configures the various components for you. You will still need to learn some command-line incantations to interact with your node, but most of the initial work is done for you.
 
 ==== RaspiBlitz
 
-One of the most popular and complete "helper" is _RaspiBlitz_, a project built by Christian Rotzoll. It is intended to be installed on a Raspberry Pi 4. RaspiBlitz comes with a recommended hardware "kit" that you can build in a matter of hours or at most a weekend. If you attend a Lightning "hackathon" in your city, you are likely to see many people working on their RaspiBlitz setup, swapping tips, and helping each other. You can find the RaspiBlitz project here:
+One of the most popular and complete "helpers" is _RaspiBlitz_, a project built by Christian Rotzoll. It is intended to be installed on a Raspberry Pi 4. RaspiBlitz comes with a recommended hardware "kit" that you can build in a matter of hours or at most a weekend. If you attend a Lightning "hackathon" in your city, you are likely to see many people working on their RaspiBlitz setup, swapping tips, and helping each other. You can find the RaspiBlitz project here:
 
 https://github.com/rootzoll/raspiblitz
 
 image::images/raspiblitz.jpg[]
 
 In addition to a Bitcoin and Lightning node, RaspiBlitz can install a number of additional services, such as:
 
-* TOR (Run as Hidden Service)
```

Comment:
The capitalization here is clearly copied from the RaspiBlitz website.  My suggested changes are more accurate.